initfiles/sbin/hpcc_setenv.in

```diff
@@ -81,6 +81,7 @@ IFS="${OIFS}"
 PATH_PREFIX=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^path *= *" | sed -e 's/^path *= *//'`
 
 export PID=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^pid *= *" | sed -e 's/^pid *= *//'`
+export LD_PRELOAD=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^java *= *" | sed -e 's/^java *= *//'`:$LD_PRELOAD
```

Comment:
I just double checked to make sure.  Having just "java=" in the conf file has no adverse effects.  The result will be ":" or ":/whatever/ld_preload/was.so" and both are handled fine by the system.